modules/cart/src/Plugin/QueueWorker/CartExpire.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Drupal\commerce_cart\Plugin\QueueWorker;
+
+use Drupal\commerce_order\Entity\OrderInterface;
+use Drupal\Core\Queue\QueueWorkerBase;
+
+/**
+ * Removes an expired Cart.
+ *
+ * @QueueWorker(
+ *  id = "cart_expiries",
+ *  title = @Translation("Commerce Cart expiration"),
+ *  cron = {}
+ * )
+ */
+class CartExpire extends QueueWorkerBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function processItem($data) {
+    $orders = \Drupal::entityTypeManager()->getStorage('commerce_order')
```

Comment:
Let's implement the ContainerFactoryInjectionInterface, or whatever it's called. This allows us to properly inject the service. If you're not too sure, we have an example somewhere within the module.